Source/Auth/AuthManager.swift

```diff
@@ -160,21 +160,21 @@ class AuthManager {
         return nil
     }
     
-    private func refreshAccessTokenWithExpirationBuffer(bufferInMinutes: Int) -> Bool {
+    private func refreshAccessTokenWithExpirationBuffer(_ bufferInMinutes: Int) -> Bool {
```

Comment:
Understood. We need to do a refactor against these internal function to conform to new naming convention.